### ISSUE TITLE
The imposed limit on the number of characters when reading text from the clipboard using NVDA+C should be configurable.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3608]
**Reported by laughingthunder on 2013-10-26 20:46**
I suspect this limit was imposed during the early days before NVDA was truly a stable, viable option for screen reading on Windows.  I happen to use this command often and feel that the character limit is arbitrary and unnecessary.  Perhaps it should be configurable by the user, since I can see how it may be helpful to limit the number of characters to be spoken in cases where the command is accidentally pressed.  I also understand that this command may also be one that is not used often.  However, I feel that it should be configurable for those of us who make regular use of it.

For example, I find the command to be very useful in situations where text is only easily accessible using the mouse, such as on extremely complex web sites.  In such cases, I am able to simply locate the text with NVDA’s mouse echo feature, highlight it using my remaining vision, and place it on the clipboard.  I then use the clipboard as a kind of buffer for reading text without needing to remember navigation commands or get stuck navigating around large blocks of links or unlabeled elements.  However, the limit on the number of characters becomes an issue in such cases, making this navigation work-around unnecessarily difficult.  Therefore, I feel that the limit should either become configurable, or be removed entirely.

**Comment by ehollig** [comment]
This is being discussed in #5573. Closing this in favor of #5573
